### DISCUSSION TITLE
Can I delete generate-rss.ts in the contentlayer branch?

### DISCUSSION THREAD

**Discussion by nelsonlaidev** [timlrx/tailwind-nextjs-starter-blog#405] [Q&A] [answered]
Where is the lib/generate-rss.ts used in the contentlayer branch? Can I delete it? Because I see that generate-rss.mjs already exists in scripts folder and is being used.

**Comment by timlrx** [comment, marked as answer]
Yes, it's no longer needed with contentlayer. Feel free to delete it.

**Reply by nelsonlaidev** [reply]
Thank you! ❤️
